src/plugins/remove/azureRemovePlugin.ts

```diff
@@ -3,10 +3,37 @@ import { ResourceService } from "../../services/resourceService";
 import { AzureBasePlugin } from "../azureBasePlugin";
 
 export class AzureRemovePlugin extends AzureBasePlugin {
-  public hooks: { [eventName: string]: Promise<any> };
 
   public constructor(serverless: Serverless, options: Serverless.Options) {
     super(serverless, options);
+
+    this.commands = {
+      remove: {
+        usage: "Remove service resource group (USE WITH CAUTION)",
```

Comment:
eeep, is there a way to enforce confirmation with this command?

Reply:
Not currently, no